snapshot/db.go

```diff
@@ -4,43 +4,28 @@ import (
 	"github.com/scootdev/scoot/snapshot/git/repo"
 )
 
-// ID identifies a Value in DB. (Cf. doc.go for explanation of Value) Opaque to the client.
+// ID identifies a Snapshot in DB. (Cf. doc.go for explanation of Snapshot) Opaque to the client.
 type ID string
 
-// DB is the Scoot Database, allowing creation, distribution, and export of Values.
-type DB interface {
+// Creator allows creating new Snapshots.
+type Creator interface {
 	// Ingest
 
 	// IngestDir ingests a directory directly.
```

Comment:
these comments are defining the terms with the term, eg. IngestDir ingests a directory... can we modify it to define ingest?  eg: IngestDir creates a new snapshot containing a copy of the files in dir.  ?The root of the snapshot will be dir?  

Also - what is FSSnapshot?  where can I find the definition for this?

This comment also applies to the definition of IngestCommit